source/routes/delete/daily.js

```diff
@@ -0,0 +1,37 @@
+const PouchDB = require('pouchdb');
```

Comment:
This is from your daily CRUD, should definitely not be in here and should be removed.